docs/Running-Spark-on-Tachyon.md

```diff
@@ -1,6 +1,8 @@
 ---
 layout: global
 title: Running Spark on Tachyon
+nickname: Spark
```

Comment:
Is this the title that shows up in the navbar? If so, then Apache projects should be prefixed with "Apache".